apps/plea/email.py

```diff
@@ -43,12 +43,15 @@ def send_plea_email(context_data):
 
     context_data: dict populated by form fields
     """
-    try:
-        court_obj = Court.objects.get_by_urn(context_data["case"]["urn"])
-    except Court.DoesNotExist:
-        logger.error("URN does not have a matching Court entry: {}".format(
-            context_data["case"]["urn"]))
-        raise
+
+    # Get or create case
```

Comment:
OK, get_or_create is no good here as we commonly have a situation where there are multiple items.  I've refactored to use exceptions and be more pythonic.